politeiad/cmd/politeia/politeia.go

```diff
@@ -58,12 +68,20 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "  inventory         - Inventory records "+
 		"<vetted count> <branches count>\n")
 	fmt.Fprintf(os.Stderr, "  new               - Create new record "+
-		"<name> <filename>...\n")
+		"[mdid:metadata]... <filename>...\n")
 	fmt.Fprintf(os.Stderr, "  getunvetted       - Retrieve record "+
 		"<id>\n")
 	fmt.Fprintf(os.Stderr, "  setunvettedstatus - Set unvetted record "+
 		"status <publish|censor> <id>\n")
-	//fmt.Fprintf(os.Stderr, "  update      - Update record\n")
+	fmt.Fprintf(os.Stderr, "  update            - Update unvetted record "+
+		"[actionmdid:metadata]... <actionfile:filename>...\n")
+	fmt.Fprintf(os.Stderr, "\n")
+	fmt.Fprintf(os.Stderr, " mdid is a number. Example with 2 metadata records "+
+		"metadata0:{\"moo\":\"12\",\"blah\":\"baz\"} "+
+		"metadata1:{\"lala\":42}\n")
+	fmt.Fprintf(os.Stderr, " actionmdid is an action + metadatastream id "+
```

Comment:
There doesn't seem to be a way to delete metadata streams. Is this a gap in implementation or by design?

Reply:
"" overwrites it and is, therefore, the same as a delete.  This is by design.